app/serializers/application.js

```diff
@@ -38,17 +38,37 @@ export default HalSerializer.extend({
   },
 
   normalize: function(type, hash, property) {
-    var payload = this._super(type, hash, property);
+    var payload = this._super(...arguments);
 
     if (payload.links && payload.links.account) {
       payload.links.stack = payload.links.account;
       delete payload.links.account;
     }
+
     if (payload.links && payload.links.accounts) {
```

Comment:
These two `if (payload.links && ....)` sections can be moved into the `if(payload.links)` block below and drop the extra conditional.